nilearn/maskers/nifti_labels_masker.py

```diff
@@ -202,6 +203,32 @@ def __init__(
 
         self.background_label = background_label
         self._original_region_ids = self._get_labels_values(self.labels_img)
+        self._region_id_name = None
+
+        # if the labels is a path, it is the path to tsv file, so read it
+        if isinstance(labels, str):
+            try:
```

Comment:
I think I would not go for a `try / except` here but maybe there is a reason I am missing.
was it to provide more meaningful error message than pandas' "file not found" ?

Reply:
It's because instead of throwing an error and stopping the code, I thought maybe it's better to throw a warning and set `self.labels` to `None`.